araproc/analysis/standard_reco.py

```diff
@@ -96,7 +96,7 @@ def __init__(self,
         # so look that up
 
         self.calpulser_r_library = {
-            100 : "49.48",
+            100 : "48.02",
```

Comment:
Is this intended? It keeps changing....

Reply:
Yes, this is restoring the value that was changed in [PR#42](https://github.com/ara-software/AraProc/pull/42). I did this based on the values Alex said to ignore/take seriously.

Reply:
Okay, great